package NDoom;

message TMegaWadInfoData {
    message TLumpInfo {
        required string Type = 1;
        required uint32 Index = 2;
    }

    repeated TLumpInfo GlobalLumps = 1;
    repeated string DocLumps = 2;
    required uint32 FirstDoc = 3;
    required uint32 DocCount = 4;
}
